docs/rst/tutorials/core/deployment/keys/filtered_topic_keys.rst

```diff
@@ -0,0 +1,188 @@
+    wget -O src/filtered_keyed_sensor.cpp https://raw.githubusercontent.com/eProsima/vulcanexus/{DISTRO}/docs/resources/tutorials/core/deployment/keys/demo_keys_filtering_cpp/src/filtered_keyed_sensor.cpp
+    wget -O src/filtered_keyed_controller.cpp https://raw.githubusercontent.com/eProsima/vulcanexus/{DISTRO}/docs/resources/tutorials/core/deployment/keys/demo_keys_filtering_cpp/src/filtered_keyed_controller.cpp
+    wget -O launch/keyed_sensors_launch.py https://raw.githubusercontent.com/eProsima/vulcanexus/{DISTRO}/docs/resources/tutorials/core/deployment/keys/demo_keys_filtering_cpp/launch/keyed_sensors_launch.py
+
+The resulting directory structure should be:
+
+.. code-block::
+
+    ~/vulcanexus_ws
+     ├──src
+        ├── demo_keys_filtering_cpp
+            ├── CMakeLists.txt
+            ├── README.md
+            ├── launch
+            │   └── keyed_sensors_launch.py
+            ├── msg
+            │   └── KeyedSensorDataMsg.idl
+            ├── package.xml
+            └── src
+                ├── filtered_keyed_controller.cpp
+                └── filtered_keyed_sensor.cpp
+
+A brief analysis on the provided files is explained below:
+
+* *demo_keys_filtering_cpp* : This directory likely contains the main source code and configuration files for the demonstration.
+* ``CMakeLists.txt``: This file is used with CMake to specify build configurations and dependencies.
+* ``README.md``: This is likely a markdown file providing instructions or information about the demonstration.
+* *launch*: This directory probably contains launch configuration files for launching ROS nodes.
+  * ``keyed_sensors_launch.py``: This Python script is likely used to launch the demonstration nodes.
+* *msg*: This directory contains message definition files.
+  * KeyedSensorDataMsg.idl: This is likely an IDL (Interface Definition Language) file defining the message structure for keyed sensor data.
+* ``package.xml``: This is an XML file containing metadata about the ROS package.
+* *src*: This directory contains the source code files for the demonstration.
+  * ``filtered_keyed_controller.cpp``: This is likely the source code for a controller node that filters keyed sensor data in reception.
+  * ``filtered_keyed_sensor.cpp``: This is likely the source code for a sensor node that publishes keyed sensor data.
+
+Building the demo package
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once the environment has been setup and the demo package sources are available, we are ready to build the workspace.
+Get into the root of the workspace and build it with the following commands:
+
+.. code-block:: bash
+
+    source /opt/vulcanexus/{DISTRO}/setup.bash
+    cd ~/vulcanexus_ws
+    colcon build
+
+
+Running the demo
+----------------
+
+In the demo, different sensors are publishing data
+Run the demo by executing the following commands in separate terminals:
```

Comment:
I think we need a bit more explanation than this. Again a small diagram would be lovely